guideline/cfh.md

```diff
@@ -131,26 +131,11 @@ off by the PAC before they can be put in place.
    Proposals will have to provide details about how these can be implemented at
    the proposed venues.
 
-   1. Food and drinks for lunches and breaks on all conference days (and
```

Comment:
USD 150 is too much.  The admission (early bird individual rate) of the past PyCon APAC's didn't reach that number.  I think it's the team's responsibility to bring down that cost.  Even in downtown Manhattan one can feed himself/herself with less than 100 bucks.